src/Symfony/Component/HttpFoundation/File/UploadedFile.php

```diff
@@ -141,9 +141,26 @@ public function guessClientExtension()
      * It is extracted from the request from which the file has been uploaded.
      * Then it should not be considered as a safe value.
      *
-     * @return int|null The file size
+     * @deprecated since 4.1 will be removed in 5.0 use getFileSize instead.
+     *
+     * @return int|null The file sizes
      */
     public function getClientSize()
+    {
+        @trigger_error(sprintf('%s is deprecated since 4.1 and will be removed in 5.0 use %s instead.', __METHOD__, 'getFileSize'), E_USER_DEPRECATED);
```

Comment:
We really want the first `%s` to be quoted i guess for HTML purpose (e.g. make it bold in the profiler log panel).

Also i suggested to start a new sentence as of `5.0. Use ... instead` (no real reason to use a sprintf placeholder here).

Sorry :) just following 3.x convention AFAIK.